File: Website/app/format/format.js
app.controller('formatController', function($scope, $location, $analytics, tsonService) {
	$analytics.pageTrack($location.path());
	$scope.result = null;

	$scope.submitForm = function() {
			$scope.result = null;
			tsonService.format($scope.tson)
				.success(function(data) {
					$scope.result = data;
				})
				.error(function(data) {
					$scope.result = data;
				});
		};
});

File: Api/ApiService/ServiceModels/Format.cs
﻿using System;
using ServiceStack;

namespace TsonService
{
    [Route("/format", "POST")]
    public class Format
    {
        public string Tson { get; set; }
    }

    public class FormatResponse
    {
        public string Tson { get; set; }
        public ErrorInfo Error { get; set; }
        public ResponseStatus ResponseStatus { get; set; }
    }
}

